.github/workflows/promote-ga.yml

```diff
@@ -29,6 +29,7 @@ jobs:
       - name: "make release/promote-oss/to-ga"
         run: |
           make release/promote-oss/to-ga
+      - uses: ./.github/actions/end-job
```

Comment:
Shouldn't this be `after-job`? or even better `post-job`?

Reply:
I didn't like `post` because `post` can be a verb.  But yeah, it shouldn't be `end-job`.

Reply:
Totally OK with `after-job` if the directory name gets synced up. 🙂